easybuild/easyconfigs/p/PhyloPhlAn/PhyloPhlAn-3.0.2-foss-2021a.eb

```diff
@@ -0,0 +1,46 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+# update to v3.0.2
+
+easyblock = 'PythonPackage'
+
+name = 'PhyloPhlAn'
+version = '3.0.2'
+
+homepage = 'https://github.com/biobakery/phylophlan'
+description = """
+PhyloPhlAn is an integrated pipeline for large-scale phylogenetic profiling of
+genomes and metagenomes. PhyloPhlAn is an accurate, rapid, and easy-to-use method
+for large-scale microbial genome characterization and phylogenetic analysis
+at multiple levels of resolution.
+"""
+
+source_urls = ['https://github.com/biobakery/%(namelower)s/archive/refs/tags']
+sources = ['%(version)s.tar.gz']
+checksums = ['c342116662bbfbb49f0665291fc7c0be5a0d04a02a7be2da81de0322eb2256b4']
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
```

Comment:
Thanks for your contribution. I see that you already added all the dependencies of PhyloPhlAn, but apparently it also needs some external tools to work (see https://github.com/biobakery/phylophlan/wiki#external-tools). I have no experience with PhyloPhlAn, should we also include a minimum set of those externals tools as dependencies here or can PhyloPhlAn be used without them?

Reply:
> Thanks for your contribution. I see that you already added all the dependencies of PhyloPhlAn, but apparently it also needs some external tools to work (see https://github.com/biobakery/phylophlan/wiki#external-tools). I have no experience with PhyloPhlAn, should we also include a minimum set of those externals tools as dependencies here to be able to use PhyloPhlAn, or does it provide some functionality without them? We already have easyconfigs for most of those external tools, so adding a minimum set here should be easy.

@lexming done